docs/source/community/build_ci_governance.rst

```diff
@@ -0,0 +1,20 @@
+PyTorch Governance | Build + CI
+===============================
+
+How to Add a New Maintainer
+---------------------------
+
+For the person to be a maintainer, a person needs to:
+
+* land at least six commits to the related part of the PyTorch repository
+* at least one of these commits must be submitted in the last six months
+
+To add a qualified person to the maintainers' list, please create
+a PR that adds a person to the `persons of interests <https://pytorch.org/docs/master/community/persons_of_interest.html>`__ page and
+`merge_rules <https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.json>`__ files. Current maintainers will cast their votes of
+support. Decision criteria for approving the PR:
+* Not earlier than two business days passed before merging (make
+  sure that the majority of the contributors have seen it)
+* PR has the correct label (`module: ci`)
+* There are no objections from the current maintainers
+* There are at least three *thumbs up* from current maintainers
```

Comment:
Might be useful to add a criteria for a "maintainer" to be demoted to "emeritus".

E.g. "If at least six months go by without a maintainer making contributions to pytorch, then their status will be changed to the honorary emeritus"